app/parsers/hyku_addons/csv_parser.rb

```diff
@@ -67,5 +67,28 @@ def collections_total
     def path_to_files
       ENV['BULKRAX_FILE_PATH'].presence || super
     end
+
+    # All possible column names
+    def export_headers
+      # Trust that the entries' parsed metadata
+      importerexporter.entries.where(identifier: current_work_ids)[0..limit || total].collect { |e| e.parsed_metadata.keys }.flatten.uniq
```

Comment:
Can we use ActiveRecords `limit` and avoid getting all docs from the DB to discard them later in memory?
```suggestion
      importerexporter.entries.where(identifier: current_work_ids).limit(limit || total).map { |e| e.parsed_metadata.keys }.flatten.uniq
```